Visitors chat

damencho opened the pull request:
<!--
Thank you for your pull request. Please provide a thorough description below.

Contributors guide: https://github.com/jitsi/jitsi-meet/blob/master/CONTRIBUTING.md
-->

damencho commented:
I will need to rebase this one, once https://github.com/jitsi/jitsi-meet/pull/13110 is merged, as there is a small conflict in main.json.

bgrozev commented on resources/prosody-plugins/mod_fmuc.lua:
```diff
@@ -236,3 +239,44 @@ end
 end);
+
+-- only live chat is supported for visitors
+module:hook("muc-occupant-groupchat", function(event)
```
Do we have anything that blocks private messages sent by visitors?

damencho replied:
Nope, just UI is hidden. I will take a look.

damencho commented:
I have addressed the comments, but there is a UI bug around grouping the messages, where all visitor messages come from the same participant. I will be looking at that tomorrow.
